apax/md/ase_calc.py

```diff
@@ -9,8 +9,8 @@
 from ase.calculators.calculator import Calculator, all_changes
 from ase.calculators.singlepoint import SinglePointCalculator
 from flax.core.frozen_dict import freeze, unfreeze
-from matscipy.neighbours import neighbour_list
 from tqdm import trange
+from vesin import NeighborList
```

Comment:
`vesin` > `matscipy`. Do we still need `matscipy` elsewhere? I think it can be removed then.

Reply:
I think I reviewed before you made the changes to pyproject.toml